### ISSUE TITLE
Cyrillic font is broken in some cases

### ISSUE THREAD

**Issue by chomarian** [tomahawk-player/tomahawk#546]
Some tracks with Russian titles are displayed with broken codepage, however some are fine as shown on the screenshot. In a folder file names are correct. (Win10, Tomahawk 0.8.4)

![bug](https://monosnap.com/file/Ge2D8ztb7pOXGeZHlf8i3TYZIPcypY.png)

**Comment by theli-ua** [comment]
tbh it just looks like tags do not have a proper encoding

**Comment by chomarian** [comment]
@theli-ua yes, the reason seems to be in tags' encoding, though I consider it as a bug since I have to solve the problem with some mp3 batch tag editor to continue using Tomahawk. On the other hand MediaMonkey player works with all these files correctly without any additional manipulations.
